js/bezirkssite-hide-empty-events.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Hide events block on bezirkssites if no events exist
+ */
+
+(function () {
+    const events_container = document.querySelector('.bg-events-container');
+    if (events_container != null && events_container.childElementCount === 0) {
+        events_container.closest('.wp-block-group').classList.add('bg-events-hidden');
+    }
```

Comment:
Das ist blöd dass das nicht mit CSS geht 🙄 
Kannst du das zu einem anderen JS File dazugeben damit wir wenigstens den zusätzlichen Request sparen?

Reply:
Ok habs mit form-hidden.js zusammen geschmissen in ein bezirk-helper.js